app/models/foreman_resource_quota/resource_quota.rb

```diff
@@ -26,6 +26,10 @@ class ResourceQuota < ApplicationRecord
     scoped_search on: :name, complete_value: true
     scoped_search on: :id, complete_enabled: false, only_explicit: true, validator: ScopedSearch::Validators::INTEGER
 
+    def self.permission_name
+      'resource_quotas'
+    end
```

Comment:
What is this doing?

Reply:
Since we adapt the permission naming to be consistent with other plugins, we also have to set a custom permission name.

It comes into action [here](https://github.com/theforeman/foreman/blob/develop/app/services/association_authorizer.rb#L17).

And Salt does the same [here](https://github.com/search?q=repo%3Atheforeman%2Fforeman_salt%20self.permission_name&type=code).